File: prime.py
#Prime numbers from 1 to 100

def isPrime(n):
    if n <= 1:
        return False
    for i in range(2, n):
        if n%i == 0:
            return False
    return True
    
start = 1
destiny = 100
for i in range(start, destiny+1):
    if isPrime(i):
        print(i)

File: prime.java
//Prime numbers from 1 to 20

import java.util.Scanner;
public class Main {
    static boolean isPrime(int n){
        if(n<=1){
            return false;
        }
        for(int i = 2; i < n; i++){
            if(n%i == 0){
                return false;
            }
        }
        return true;
    }
    public static void main(String[] args) {
        int start = 1;
        int destination = 20;
        for(int i = start; i < destination; i++){
            if(isPrime(i)){
                System.out.println(i);
            }
        }
    }
}
